packages/odata-common/src/request/odata-get-by-key-request-config.spec.ts

```diff
@@ -1,9 +1,8 @@
 import { v4 as uuid } from 'uuid';
 import { TestEntity } from '@sap-cloud-sdk/test-services/v2/test-service';
-import { oDataUri as oDataUriV2 } from '@sap-cloud-sdk/odata-v2';
 import { ODataGetByKeyRequestConfig } from '@sap-cloud-sdk/odata-common';
-import { testEntityResourcePath } from '../../../../test-resources/test/test-util/test-data';
-import { uriConverter } from '../../../odata-v2/src/uri-conversion/uri-value-converter';
+import { uriConverter, oDataUri as oDataUriV2 } from '@sap-cloud-sdk/odata-v2/';
```

Comment:
I feel this should be moved to odata-v2 package.
The same applies to other tests in this folder.
Should we create a follow up?

Reply:
We have already the ticket #98 for that.